pyanaconda/packaging/dnfpayload.py

```diff
@@ -98,6 +99,11 @@ def _df_map():
         if not key.startswith('/'):
             continue
         structured[key] = Size(int(val)*1024)
+
+    # Add /var/tmp/ if this is a directory or image installation
+    if flags.dirInstall or flags.imageInstall:
+        var_tmp = os.statvfs("/var/tmp")
+        structured["/var/tmp"] = Size(var_tmp.f_frsize * var_tmp.f_bfree)
```

Comment:
Is there (somewhere further in the code) some check testing that the directory is writable?